src/views/AnnotationView.js

```diff
@@ -1,21 +1,24 @@
 import React from 'react';
 import FileEditorContainer from '../containers/FileEditorContainer';
+import {GridContainer} from 'kf-uikit';
 
 const AnnotationView = ({match}) => {
   // The kf_id of the file
   const fileId = match.params.fileId;
   return (
-    <div className="sm:px-20 p-2 BodyContent">
-      <h3 className="text-blue font-normal my-16">Tell us about your files</h3>
-      <p className="pb-20">
+    <GridContainer collapsed="cells" className="my-20 px-12">
+      <h3 className="text-blue font-normal m-0 cell-12 row-1">
+        Tell us about your files
+      </h3>
+      <p className="m-0 cell-8 row-2">
```

Comment:
Loving the file editor view on mobile, just need to make sure the help text goes full width on small screens 

--- 
`320px wide`
<img width="313" alt="Screen Shot 2019-04-30 at 1 05 56 PM" src="https://user-images.githubusercontent.com/1334131/56979928-fdb7a880-6b48-11e9-9274-bd974d5c8dd2.png">